source/features/index.tsx

```diff
@@ -59,21 +59,33 @@ const logError = (url: string, error: unknown): void => {
 	// Don't change this to `throw Error` because Firefox doesn't show extensions' errors in the console
 	console.group('❌', id, version, pageDetect.isEnterprise() ? 'GHE →' : '→', error);
 
+	const searchIssueUrl = new URL('https://github.com/refined-github/refined-github/issues');
+	searchIssueUrl.searchParams.set('q', `is:issue is:open sort:updated-desc "${message}"`);
 	console.group('Search issue');
-	console.log(`https://github.com/refined-github/refined-github/issues?q=is%3Aissue+${encodeURIComponent(message)}`);
+	console.log(searchIssueUrl.href);
 	console.groupEnd();
 
-	const newIssueUrl = new URL('https://github.com/refined-github/refined-github/issues/new?labels=bug&template=1_bug_report.md');
+	const newIssueUrl = new URL('https://github.com/refined-github/refined-github/issues/new');
+	newIssueUrl.searchParams.set('labels', 'bug');
+	newIssueUrl.searchParams.set('template', '1_bug_report.yml');
 	newIssueUrl.searchParams.set('title', `\`${id}\`: ${message}`);
-	newIssueUrl.searchParams.set('body', stripIndent(`
+	newIssueUrl.searchParams.set('description', stripIndent(`
 		<!-- Please also include a screenshot if the issue is visible -->
 
-		URL: ${location.href}
-
 		\`\`\`
 		${error instanceof Error ? error.stack! : error as string}
 		\`\`\`
 	`));
+	newIssueUrl.searchParams.set('example_url', location.href);
+
+	const browser = /(Chrome|Firefox|Safari)\/\d+/.exec(navigator.userAgent);
+	// Unknown browsers will need to be specified manually
+	if (browser) {
+		// @ts-expect-error navigator.userAgentData is not typed
+		const {platform} = navigator.userAgentData || navigator;
+		newIssueUrl.searchParams.set('browser_s_used', `${platform as string} ${browser[0]}`);
+	}
```

Comment:
As [mentioned before](https://github.com/refined-github/refined-github/pull/5113#discussion_r759351014), but we should not do this. Let the user enter the browser, which may be reported as "chrome" when in reality it's Vivaldi. We don't need to parse this information because it's so fickle 

```suggestion
```